benchmarks/Makefile

```diff
@@ -1,11 +1,16 @@
+-include ../include/conv-mach-opt.mak
+
 # Temporary hack that needs to be removed / replaced
```

Comment:
What is the 'temporary hack' referring to now? (also in the other files)

Reply:
`OPTS += $(OPTSATBUILDTIME)`